src/OrbitQt/orbitsamplingreport.cpp

```diff
@@ -93,7 +94,38 @@ void OrbitSamplingReport::Initialize(orbit_data_views::DataView* callstack_data_
     //  is no need for manual updates.
     orbit_data_views_.push_back(treeView);
 
-    QString thread_name = QString::fromStdString(report_data_view.GetName());
+    std::string thread_name_std = report_data_view.GetName();
+    QString thread_name = QString::fromStdString(thread_name_std);
+
+    uint32_t thread_id = report_data_view.GetThreadID();
+    // Report any thread that contains more than 5% unwinding errors.
+    constexpr double kUnwindErrorThreshold = 0.05;
+    double unwind_error_ratio = sampling_report_->UnwindErrorRatio(thread_id);
+    if (sampling_report_->UnwindErrorRatio(thread_id) > kUnwindErrorThreshold) {
+      auto* notice_box = new QWidget(tab);
+      notice_box->setStyleSheet(
+          ".QWidget { border-radius: 5px; border: 1px solid palette(text); background:"
+          " rgba(255,0,0, 10%); } "
+          "QPushButton:focus { border-color: rgba(255, 0, 0, 20%); }");
+      auto* notice_box_layout = new QGridLayout(notice_box);
+
+      auto* notice_label =
+          new QLabel(QString::fromStdString(absl::StrFormat(
+                         "There are %.2f %% unwinding errors in %s.", unwind_error_ratio * 100,
+                         std::regex_replace(thread_name_std, std::regex("\n"), " "))),
```

Comment:
Bit of a hack here with removing the new line used in the tab name. Not sure if worth refactoring this to improve it.

Reply:
Yeah, I don't like it either. But I don't think it is worth refactoring this. I'd keep it as it, unless you or @beckerhe object.